docs/index.md

```diff
@@ -1,7 +1,7 @@
 # Automata
 
-_Copyright 2016-2022 Caleb Evans_
-_Released under the MIT license_
+*Copyright 2016-2023 Caleb Evans*  
```

Comment:
```suggestion
*Copyright 2016-2024 Caleb Evans*  
```
One day early lol

Reply:
@eliotwrobson Done in c58b594c1cfc92f6721df1224c3a5b8291e0e89d—but made sure to update all the other relevant copyright years as well.